src/Computer/GlideComputerAirData.cpp

```diff
@@ -132,10 +132,9 @@ GlideComputerAirData::ProcessVertical(const MoreData &basic,
                         calculated);
   AverageClimbRate(basic, calculated);
 
-  if (calculated.circling)
+  if (calculated.circling) {
     CurrentThermal(basic, calculated, calculated.current_thermal);
-  else
-    calculated.current_thermal = calculated.last_thermal;
```

Comment:
Why .. remove this? I don't get it, and I don't get your commit message.

Reply:
All logic what to do when leaving a climb transitioning to cruise, or being in cruise mode, is hosted in LastThermalStats() except this line. I joined to the rest of the logic to have it consistent. This assignment is only needed once when transitioning to cruise mode, not every time updating while in cruise mode.